### PR TITLE
feat: UI support for invokedByDefault transformers

### PR DESCRIPTION
**Pull request by kmehant**
Signed-off-by: Mehant Kammakomati <kmehant@gmail.com>

### CODE REVIEW

**Review by ashokponkumar — DISMISSED** [review]

File: src/app/projects/ProjectPlan.tsx [hunk 1]
```diff
@@ -89,7 +79,7 @@ function ProjectPlan(props: IProjectPlanProps): JSX.Element {
                     <ToolbarContent>
                         <ToolbarItem>
                             <Button
-                                isDisabled={disableThisSection}
+                                isDisabled={isPlanning}
```
Comment (HarikrishnanBalagopal) [inline src/app/projects/ProjectPlan.tsx:82]: Keep the variable name as `disableThisSection` and just define it as `const disableThisSection = isPlanning;` somewhere above.

File: src/app/projects/ProjectPlan.tsx [hunk 2]
```diff
@@ -122,7 +112,7 @@ function ProjectPlan(props: IProjectPlanProps): JSX.Element {
                         </ToolbarItem>
                         <ToolbarItem>
                             <Button
-                                isDisabled={disableThisSection || !props.project.status?.plan}
+                                isDisabled={!props.project.status?.plan}
```
Comment (HarikrishnanBalagopal) [inline src/app/projects/ProjectPlan.tsx:115]: see above

File: src/app/projects/ProjectPlan.tsx [hunk 3]
```diff
@@ -167,7 +157,7 @@ function ProjectPlan(props: IProjectPlanProps): JSX.Element {
                     name="plan"
                     aria-label="plan"
                     resizeOrientation="vertical"
-                    isDisabled={disableThisSection || !props.project.status?.plan}
+                    isDisabled={!props.project.status?.plan}
```
Comment (HarikrishnanBalagopal) [inline src/app/projects/ProjectPlan.tsx:160]: see above

**Review by HarikrishnanBalagopal — CHANGES_REQUESTED** [review]

File: src/app/qa/MultiLine.tsx [hunk 4]
```diff
@@ -20,7 +20,7 @@ import { QAContext } from './QAContext';
 import { IQAComponentProps } from './QAWizard';
 import { TextContent, TextArea } from '@patternfly/react-core';
 
-function Multiline(props: IQAComponentProps): JSX.Element {
+function MultiLine(props: IQAComponentProps): JSX.Element {
```
[the remark this answers is not in the document]
Comment (kmehant) [reply src/app/qa/MultiLine.tsx:23]: @HarikrishnanBalagopal  @ashokponkumar 
https://github.com/konveyor/move2kube-ui/pull/128/files#diff-9fbd7a7707d6fad02d98c83cc8c72a94821fa17a1983f54f1ef9c3da59bb1d25R113 

Its `MultiLineInput` in https://github.com/konveyor/move2kube/blob/main/types/qaengine/problem.go#L42 and `Multiline` in UI due to which multi line type questions breaking the QA engine at UI.

Weed need to decide whether following `MultiSelect` should we change this to `MultiLine` or change it to `MultiLineInput` which is there in move2kube code presently.

[the remark this answers is not in the document]
Comment (kmehant) [reply src/app/qa/MultiLine.tsx:23]: In any case, this change is required for the QA engine not to break for multiline questions at UI

[the remark this answers is not in the document]
Comment (HarikrishnanBalagopal) [reply src/app/qa/MultiLine.tsx:23]: This only changes the function name, it has zero impact in terms of what gets sent over the wire.

[the remark this answers is not in the document]
Comment (kmehant) [reply src/app/qa/MultiLine.tsx:23]: Can you please check the completes changes?

[the remark this answers is not in the document]
Comment (HarikrishnanBalagopal) [reply src/app/qa/MultiLine.tsx:23]: I see the issue https://github.com/konveyor/move2kube/blob/main/types/qaengine/problem.go#L42
currently has `MultiLineInput` and the QA Wizard https://github.com/konveyor/move2kube-ui/blob/main/src/app/qa/QAWizard.tsx#L113 has `Multiline`

The only change required is to change https://github.com/konveyor/move2kube-ui/blob/main/src/app/qa/QAWizard.tsx#L113 to `MultiLineInput` to match the CLI. It also makes it similar to `Input` we already have. https://github.com/konveyor/move2kube-ui/blob/main/src/app/qa/QAWizard.tsx#L111

[the remark this answers is not in the document]
Comment (kmehant) [reply src/app/qa/MultiLine.tsx:23]: But by the semantic meaning `Input` can be anything.

[the remark this answers is not in the document]
Comment (kmehant) [reply src/app/qa/MultiLine.tsx:23]: Why not`MultiSelectInput` 🤔

**Comment by kmehant** [comment]
@HarikrishnanBalagopal Made the changes, please review!

**Comment by kmehant** [comment]
fixes https://github.com/konveyor/move2kube-ui/issues/129

**Review by HarikrishnanBalagopal — APPROVED** [review]
LGTM

**Pull request merged**